src/APIM_ARMTemplate/apimtemplate/Creator/TemplateCreators/ProductAPITemplateCreator.cs

```diff
@@ -55,7 +58,7 @@ public List<ProductAPITemplateResource> CreateProductAPITemplateResources(APICon
             string[] allDependsOn = dependsOn;
             foreach (string productID in productIDs)
             {
-                ProductAPITemplateResource productAPITemplate = this.CreateProductAPITemplateResource(productID, api.name, allDependsOn);
```

Comment:
This is our internal style, we should keep our consistency

Reply:
https://github.com/Azure/azure-api-management-devops-resource-kit/issues/615

Reply:
Ye, will do it definitely!